CMakeLists.txt

```diff
@@ -18,6 +18,7 @@ option(NETWORKIT_CLANG_TIDY "Check code with clang-tidy" OFF)
 option(NETWORKIT_FLATINSTALL "Install into a flat directory structure (useful when building a Python package)" OFF)
 option(NETWORKIT_COVERAGE "Build with support for coverage" OFF)
 option(NETWORKIT_SANITY_CHECKS "Algorithms will perform further (expensive) checks" OFF)
+option(NETWORKIT_USE_DHB "Enable DHB)" ON)
```

Comment:
`DHB` is set to enabled by default here. Does this also mean, that it is used instead of `Graph` as data structure by default?

Reply:
No it is not used by default. Instead there is a `DHBGraph` class now that can be instantiated. We did not change the default `Graph` class.

Reply:
I see. When first reading into the PR I misunderstood how `DHB` is integrated.